### ISSUE TITLE
Iframe redrawing on scroll

### ISSUE THREAD

**Issue by lpichoud** [ionic-team/ionic#20594]
**Ionic version:**

[x] **4.x**
[ ] **5.x**

**I'm submitting a ...** 

[x ] bug report
[ ] feature request

**Current behavior:**
I'm displaying a youtube video iframe in ion-content.  
When I enable ionScroll event on ion-content and I scroll, the iframe is disappearing. It acts the same way as if the page was resized.

When ionScroll is not enabled there is no problem.

**Expected behavior:**

Having the iframe displayed all the time, not redrawing on scroll

**Steps to reproduce:**

Just put an iframe in ion-content, enable scrollEvents and add ionScroll.  
scrollEvents does not trigger the issue, this is ionScroll bind that triggers it.

**Related code:**
* html
```
<ion-content [scrollEvents]="true" (ionScroll)="scroll()" #ionContent>
<iframe id="ytplayer" type="text/html" width="100%" height="300"  src="https://www.youtube.com/embed/Jf92sGUe_vk?autoplay=0"  frameborder="0"/></iframe>
</ion-content>
```

* ts
```
scroll(){
//empty on purpose, issue occurs with or without code here
}
```

**Other information:**

**Ionic info:** 

```
Ionic:

   Ionic CLI                     : 5.2.5 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.11.10
   @angular-devkit/build-angular : 0.803.25
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.1.2

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res : not installed
   native-run  : 0.2.8 (update available: 0.3.0)

System:

   NodeJS : v12.13.1 (/home/lpichoud/.nvm/versions/node/v12.13.1/bin/node)
   npm    : 6.12.1
   OS     : Linux 4.15
```

**Comment by juanmiguelbesada** [comment]
related to #20582?

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you provide a repo with the code required to reproduce this issue? I am unable to reproduce the issue using the code snippet provided.

**Comment by lpichoud** [comment]
@liamdebeasi thanks for your answer.

I tried to build a stackblitz to reproduce but wasn't able to get the issue too.
After some investigations I found that the issue wasn't due to scroll but the way I display my iframe.

Wrong way :  
`<div [innerHTML]="sanitizer.bypassSecurityTrustHtml(videoPlayer)"></div>`

Right way :  
`<iframe [src]="sanitizer.bypassSecurityTrustResourceUrl(videoUrl)"></iframe>`

With innerHTML the whole iframe is redrawn on resize / scroll.

I hope that will help someone.
